lib/IRGen/GenConstant.cpp

```diff
@@ -0,0 +1,131 @@
+//===--- GenConstant.cpp - Swift IR Generation For Constant Values --------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file implements IR generation for constant values.
```

Comment:
I just copied this structure from another file, and it seems most of the files here don't seem to use that convention (in fact, out of the ~850 .cpp and.h files in swift/lib and swift/include, only 24 contain \file).  I'll definitely change it if you think it's the right thing to do.